mmv1/third_party/terraform/services/compute/resource_compute_network_firewall_policy_rule_test.go.erb

```diff
@@ -253,6 +431,8 @@ resource "google_compute_network_firewall_policy_rule" "fw_policy_rule1" {
   security_profile_group = "//networksecurity.googleapis.com/${google_network_security_security_profile_group.security_profile_group_updated.id}"
   direction              = "INGRESS"
   disabled               = false
+  tls_inspect            = true
```

Comment:
Lets remove this

Reply:
If we remove both `tls_inspect` field the check will complain that is missing coverage

Reply:
Got it... I'll remove then!